config/v1/0000_10_config-operator_01_apiserver.crd.yaml

```diff
@@ -71,14 +71,17 @@ spec:
                       like 'Default', but logs request and response HTTP payloads
                       for write requests (create, update, patch). - AllRequestBodies:
                       like 'WriteRequestBodies', but also logs request and response
-                      HTTP payloads for read requests (get, list). \n If unset, the
+                      HTTP payloads for read requests (get, list). - UserRequests:
+                      logs resources requested bu users in the `system:authenticated:oauth` group.
```

Comment:
... and oauthaccesstoken creations and deletions.

Reply:
or a bit shorter: ... and oauthaccesstokens

Reply:
Good rec 👍 I changed it.
Incidentally... I ran make update-scripts and make generate-with-container and one of these overwrote this file.So I put the changes back in.